lib/cfg/instruction-sets/base.ts

```diff
@@ -30,12 +32,12 @@ export enum InstructionType {
 }
 
 export class BaseInstructionSetInfo {
-    static get key() {
+    static get key(): 'base' | InstructionSet | InstructionSet[] {
         return 'base';
     }
 
     isJmpInstruction(x: string) {
-        return x.trim()[0] === 'j' || x.match(/\bb\.*(eq|ne|cs|hs|cc|lo|hi|ls|ge|lt|gt|le|rge|rlt)?\b/);
+        return x.trim()[0] === 'j' || !!x.match(/\bb\.*(eq|ne|cs|hs|cc|lo|hi|ls|ge|lt|gt|le|rge|rlt)?\b/);
```

Comment:
is this still needed when there's now a specific arm32/aarch64 override? or are there other architectures that have 'branch' instructions?

Reply:
Good question, I am not sure. It might be good to leave them as a sensible default and we can override as needed.